docs/core/reference/corectl.md

```diff
@@ -141,3 +141,45 @@ other configuration will be deleted.
 ```
 corectl reset
 ```
+
+### `grant`
```

Comment:
There's a table of contents at the top of the file to add these functions to as well.